frontend/src/lib/components/flows/content/FlowModule.svelte

```diff
@@ -161,6 +172,20 @@
 												/>
 											{/if}
 										</TabContent>
+										<TabContent value="settings">
+											{#if ('path' in flowModule.value && flowModule.value.path) || ('language' in flowModule.value && flowModule.value.language)}
+												<input
+													on:click|stopPropagation={() => undefined}
+													class="overflow-x-auto"
+													type="text"
+													bind:value={flowModule.summary}
+													placeholder="Summary"
+												/>
+											{/if}
+										</TabContent>
```

Comment:
let's remove, we can already set it in 2 places normally. In the header of the module and in the minimap

Reply:
Done